docs/src/advanced/assembly.md

```diff
@@ -23,6 +23,27 @@ pub fn add_1(num: u32) -> u32 {
 }
 ```
 
+An `asm` block can only return a single register. If you really need to return more than one value, you can return a tuple. Here's an example showing how you might return a tuple of `(u64, u64)`:
```

Comment:
Should we mention that what we're really returning is a reference to a tuple and not the tuple itself? Someone might  say: "you just said I can only return a single register from an `asm` block and now you're telling me I can return a tuple that contains multiple registers?"

Reply:
👍